$OpenBSD: patch-doc_sphinx_reference_varnishd_rst,v 1.1 2018/12/14 21:22:38 kn Exp $

https://github.com/varnishcache/varnish-cache/commit/bdc0c48f38c8789337896d913e7825d0f26bc821
Bad documentation leads to bugs, bugs lead to downtime,
downtime leads to frustration.

Index: doc/sphinx/reference/varnishd.rst
--- doc/sphinx/reference/varnishd.rst.orig
+++ doc/sphinx/reference/varnishd.rst
@@ -57,7 +57,7 @@ Basic options
 
   Name is referenced in logs. If name is not specified, "a0", "a1",
   etc. is used. An additional protocol type can be set for the
-  listening socket with PROTO. Valid protocol types are: HTTP/1
+  listening socket with PROTO. Valid protocol types are: HTTP
   (default), and PROXY.
 
   Multiple listening addresses can be specified by using different
